src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+                </p>
+              </div>
+              <div class="flex align-items-center justify-content-center">
+                <PvButton
+                  class="border-circle w-6rem h-6rem m-2 surface-hover text-primary border-none font-bold flex align-items-center justify-content-center hover:text-100 hover:bg-primary"
+                  label="Show Consent"
+                  @click="seeConsent(result.consent[0])"
+                />
+              </div>
+            </div>
+          </PvFieldset>
+          <div v-else>
+            <p>No consent available.</p>
+          </div>
+        </div>
+        <div class="w-full mt-2">
+          <PvFieldset legend="Assent" v-if="consents && consents.length > 0">
+            <div class="flex flex-row w-full">
+              <div style="width: 80%">
+                <p class="m-0">
+                  <span class="font-bold">Name: </span>{{ result.assent[0]?.fileName }} <br />
+                  <span class="font-bold">Current Commit: </span>{{ result.assent[0]?.currentCommit }}
+                  <br />
+                  <span class="font-bold">GitHub Org: </span>{{ result.assent?.gitHubOrg }} <br />
+                  <span class="font-bold">GitHub Repository: </span>{{ result.assent[0]?.gitHubRepository }}
+                  <br />
+                  <span class="font-bold">Last Updated: </span>{{ result.assent[0]?.lastUpdated }} <br />
+                </p>
+              </div>
+              <div class="flex align-items-center justify-content-center">
+                <PvButton
+                  class="border-circle w-6rem h-6rem m-2 surface-hover text-primary border-none font-bold flex align-items-center justify-content-center hover:text-100 hover:bg-primary"
+                  label="Show Assent"
+                  @click="seeConsent(result.assent[0])"
+                />
+              </div>
+            </div>
+          </PvFieldset>
+          <div v-else>
+            <p>No assent available.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </PvPanel>
+  <PvDialog
+    v-if="showConsent"
+    v-model:visible="showConsent"
+    :draggable="false"
+    modal
+    :close-on-escape="false"
+    :style="{ width: '65vw' }"
+    :breakpoints="{ '1199px': '85vw', '575px': '95vw' }"
+  >
+    <div v-html="confirmText"></div>
+  </PvDialog>
+</template>
+
+<script setup>
+import { ref, onMounted, watch, computed } from 'vue';
+import { fetchLegalDocs } from '@/helpers/query/legal';
+import { useQuery } from '@tanstack/vue-query';
+import { useAuthStore } from '@/store/auth';
+import { marked } from 'marked';
+import _lowerCase from 'lodash/lowerCase';
+import _mapValues from 'lodash/mapValues';
+import _forEach from 'lodash/forEach';
+
+const props = defineProps({
+  legal: { type: Object, required: false, default: null },
+});
+
+const defaultParams = [
+  {
+    name: 'Mouse and click',
+    icon: 'pi pi-check',
```

Comment:
I feel that the icons on these default params don't provide much more context than the points themselves. What do you think about removing them all together?

Reply:
Yes, I would like to get more insightful icons, but the prime-vue version that we have does not allow, probably will keep them for now until updating